dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -164,6 +164,14 @@ def valid_scripts
     render json: scripts
   end
 
+  # GET /api/v1/sections/require_captcha
+  # Get the recaptcha site key for frontend and whether current user requires captcha verification
```

Comment:
Apologies if I'm missing something obvious, but where is the logic that determines if the current user requires captcha verification?

Reply:
The logic (which was originally introduced a couple weeks ago with that nasty bug) is on a separate branch. This PR isn't meant to implement the end-to-end workflow. Just laying the foundation -- I didn't think it was appropriate to try and reintroduce the logic yet because of HoC last week and the upcoming vacation. Seems smarter to reintroduce that after break when people are able monitor often, etc. The logic touches all aspects of joining a section, which is very highly used. I think it's advisable just to lay this groundwork now with the endpoint to serve up info from the server plus the react component. When break is over, we can try deploying the full feature. lmk if that makes sense.

Reply:
Ah, got it, thanks for the explanation!